modules/static/googlechrome/lighthouse/initref

```diff
@@ -0,0 +1 @@
+14feb91c4b8e15c06244ee673818f7fe1985bf23
```

Comment:
Waiting to know chosen initref: https://github.com/bufbuild/modules/issues/78#issuecomment-1561948041

Reply:
Just have a thought, disregard if it doesn't seem worth it. 

The latest major release (10) included some breaking changes. Would it be helpful to include the last two major releases to support the community that may not have migrated up?

Reply:
that's interesting, yeah they do semver releases, I guess we could tweak our releases syncer pkg to support regular semver and not only the wkt special case. I'll play a bit with it.

Reply:
Done, I think it's cleaner now.